CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
+- Added support for urllib3 2+ in Python 3.10+
```

Comment:
Add the PR number here like the other lines, please?

Reply:
Updated.